x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/lens_attributes.test.ts

```diff
@@ -322,15 +349,15 @@ describe('Lens Attribute', () => {
           'x-axis-column-layer0': {
             dataType: 'number',
             isBucketed: true,
-            label: 'Largest contentful paint',
+            label: 'Page load time',
```

Comment:
Why was this changed along with sourceField in this test?

Reply:
It's related to the metric options. With this change, it selects the first by default. Earlier that wasn't happening, this is why test was selecting a different metric. Now the test is updated correctly.